ksql-engine/src/main/java/io/confluent/ksql/function/udf/geo/GeoDistanceKudf.java

```diff
@@ -85,19 +109,18 @@ private void validateLatLonValues(
     }
   }
 
-  private double selectEarthRadiusToUse(final Object... args) {
+  private double selectEarthRadiusToUse(final String... radius) {
```

Comment:
```suggestion
  private double selectEarthRadiusToUse(final String radius) {
```
No need to pass in varargs here either